.coderabbit.yml

```
@@ -1,35 +1,15 @@
-    # 코딩 가이드라인 문서를 지정하는 파일 패턴입니다.
-    # CodeRabbit은 이 파일을 스캔하여 팀의 표준을 이해하고 코드 리뷰에 적용합니다.
-    # 여러 파일이 지원됩니다. 파일 이름은 대소문자를 구분합니다.
-    # 일반적인 파일들(예: .github/copilot-instructions.md)은 기본적으로 포함됩니다 [8].
-    filePatterns :
-      - "docs/team_conventions.md"   # 예시: 팀의 코딩 컨벤션 문서
+knowledge_base:
+  code_guidelines:
+    enabled: true
+    filePatterns:
+      - "docs/team_conventions.md"
+      - "**/team_conventions.md"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)